lib/mail/utilities.rb

```diff
@@ -310,7 +310,7 @@ def blank?(value)
       if value.kind_of?(NilClass)
         true
       elsif value.kind_of?(String)
-        value !~ /\S/
+        value.respond_to?(:blank?) ? value.blank? : value !~ /\S/
```

Comment:
Would make this the first conditional check rather than making it String-specific